nikola/plugins/command/new_post.py

```diff
@@ -213,6 +213,14 @@ class CommandNewPost(Command):
                     'bbcode, html, textile, txt2tags',
         },
         {
+            'name': 'kernel',
```

Comment:
>My idea would be to make it -f ipynb@python

This is interesting... I will explore this possibility... we can probably provide the more popular formats: `ipynb@python2`, `ipynb@python3`, `ipynb@julia`, `ipynb@r` and leave the others kernels as a plugable thing (I mean not in core... there are more than 30 kernel... we can not support all of them in core).